portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationAssigningRoles.testcase

```diff
@@ -729,4 +729,259 @@ definition {
+
+			Click.clickAt(
+				key_navItem = "Users",
+				locator1 = "RolesPermissionsNavigation#PERMISSIONS_NAVITEM_COLLAPSED");
+
+			AssertElementPresent(
+				key_permissionNavHeader = "Control Panel",
+				key_permissionNavItem = "Segments",
+				locator1 = "RolesPermissionsNavigation#PERMISSIONS_NAVIGATION");
+		}
+	}
+
+	@description = "LPS-167023 Given the user creates a new role
+	When the user clicks on Control Panel > Users > Segments
+	Then in General Permissions, the check Access in Control Panel permission is displayed"
+	@priority = "4"
+	test SegmentsPermissionsHaveAccessToControlPanel {
+		property custom.properties = "feature.flag.LPS-166954=true";
+
+		task ("Create a new role through Roles Admin Portlet") {
+			Role.openRolesAdmin();
+
+			Navigator.gotoNavTab(navTab = "Regular Roles");
+
+			Role.add(roleTitle = "SegmentsPermissionsHaveAccessToControlPanel Role");
+		}
+
+		task ("Define permissions and check Access in Control Panel permission is displayed") {
+			Role.editPermissionCP(
+				define = "true",
+				permissionNavHeader = "Users",
+				permissionNavItem = "Segments",
+				permissionSheetSubtitle = "Application Permissions",
+				permissionValues = "Access in Control Panel");
+
+			Role.viewPermissionCP(
+				permissionDefinitionValue = "View Control Panel Menu",
+				permissionNavigationValue = "Portal");
+		}
+	}
+
+	@description = "LPS-167023 Given the Roles portlet
+	When the user tries to add a new role
+	And the user wants to add segments permissions
+	Then the segments permissions are not allocated in Site and Asset Library Administration > People"
+	@priority = "3"
+	test SegmentsPermissionsNotAllocatedInSiteAdministration {
+		property custom.properties = "feature.flag.LPS-166954=true";
+
+		task ("Create a new role through Roles Admin Portlet") {
+			Role.openRolesAdmin();
+
+			Navigator.gotoNavTab(navTab = "Regular Roles");
+
+			Role.add(roleTitle = "SegmentsPermissionsNotAllocatedInSiteAdministration Role");
+		}
+
+		task ("Assert the segments permissions are not allocated in Site and Asset Library Administration > People") {
+			Navigator.gotoNavItem(navItem = "Define Permissions");
+
+			AssertClick(
+				key_navItem = "Site and Asset Library Administration",
+				locator1 = "RolesPermissionsNavigation#PERMISSIONS_NAVITEM_COLLAPSED",
+				value1 = "Site and Asset Library Administration");
+
+			Click.clickAt(
+				key_navItem = "People",
+				locator1 = "RolesPermissionsNavigation#PERMISSIONS_NAVITEM_COLLAPSED");
+
+			AssertElementNotPresent(
+				key_permissionNavHeader = "Site and Asset Library Administration",
+				key_permissionNavItem = "Segments",
+				locator1 = "RolesPermissionsNavigation#PERMISSIONS_NAVIGATION");
+		}
+	}
+
+	@description = "LPS-167023 Given a user without Segments permissions
+	When the user clicks on Applications Menu > Control Panel
+	Then the Segments portlet is not present and the user cannot access to the Segments page"
+	@priority = "5"
```

Comment:
Hey @CarlosBrichete , I have a doubt! When the test has the Feature Flag property, should I set the Acceptance property?

Reply:
Yes, you can set both properties

Reply:
Add the property portal.acceptance